RELEASE/scripts/autoscend.ash

```diff
@@ -1962,10 +1962,10 @@ void auto_begin()
 	// the main loop of autoscend is doTasks() which is actually called as part of the while.
 	while(doTasks())
 	{
-		if((my_fullness() >= fullness_limit()) && (my_inebriety() >= inebriety_limit()) && (my_spleen_use() == spleen_limit()) && (my_adventures() < 4) && (my_rain() >= 50) && (get_counters("Fortune Cookie", 0, 4) == "Fortune Cookie"))
+		/* if((my_fullness() >= fullness_limit()) && (my_inebriety() >= inebriety_limit()) && (my_spleen_use() == spleen_limit()) && (my_adventures() < 4) && (my_rain() >= 50) && (get_counters("Fortune Cookie", 0, 4) == "Fortune Cookie"))
```

Comment:
Is there a reason for commenting this out as opposed to just removed as fortune cookie counters are not a thing anymore?

Reply:
Not really. I just wanted a double-check, since I am not familiar with heavy rains.

Reply:
Section removed.